plugins/filter_aws/aws.h

```diff
@@ -123,4 +117,9 @@ struct flb_filter_aws {
     struct flb_filter_instance *ins;
 };
 
+struct flb_filter_aws_init_options {
+    struct flb_aws_client_generator *client_generator;
+    flb_sds_t name;
+};
+
```

Comment:
I do not understand the value provided by this, as noted in another comment.

Reply:
Sure, I've already provided the response, https://github.com/fluent/fluent-bit/pull/6065#discussion_r1006138081

Let's keep just one discussion thread open. Going to resolve this conversation.